init/kubic-init.systemd.conf

```
@@ -27,15 +35,15 @@ ExecStartPre=-/bin/sh -c '${KUBIC_INIT_RUNNER} rm kubic-init >/dev/null 2>&1'
 #       maybe the `reset` should be part of the `bootstrap` in
 #       the `kubic-init` executable, and try to 1) connect to the API server
 #       2) check the nodename exists and 3) is ready
-ExecStartPre=-/bin/sh -c 'kubectl --kubeconfig=/etc/kubernetes/admin.conf cluster-info >/dev/null 2>&1 || \
+ExecStartPre=-/bin/sh -c 'kubectl --kubeconfig=${KUBE_CFG} cluster-info >/dev/null 2>&1 || \
```

As I mentioned, using KUBECONFIG would make this argument unnecessary:

```suggestion
ExecStartPre=-/bin/sh -c 'kubectl cluster-info >/dev/null 2>&1 || \
```